### ISSUE TITLE
What is the number of points used in the PointMLP pretraining?

### ISSUE THREAD

**Issue by kaimingkuang** [salesforce/ULIP#3]
Thanks for the paper and look forward to your code! In your paper, you said 1024, 2048 and 8192 points are used for different backbones. What is the number of points for PointMLP pretraining?

**Comment by Tycho-Xue** [comment]
hi, thanks for the kind words! For modelnet40, we used 1024 points, for scanobjectnn, we reported results on both 1024 and 2048 points.

**Comment by kaimingkuang** [comment]
Thanks for the info! The learning rate schedule for pretraining is not mentioned in the paper. Did you fix it at 1e-3?

**Comment by Tycho-Xue** [comment]
we use a classic cosine schedule, lr_start=1e-6, lr_end=1e-5

**Comment by kaimingkuang** [comment]
Maybe I wasn't clear enough. What is the learning rate schedule for pretraining? The paper says it's 1e-3 but does not specify a schedule, which seems different than 1e-6 to 1e-5.
![image](https://user-images.githubusercontent.com/62201355/220234770-3a2fabea-9f85-4359-a3eb-348aa7212937.png)
